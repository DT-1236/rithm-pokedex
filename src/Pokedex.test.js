import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Pokedex from './Pokedex';

// smoke test
it('renders without crashing', () => {
  shallow(<Pokedex />);
});

// snapshot test
it('matches snapshot', () => {
  let wrapper = shallow(<Pokedex />);
  let serialized = toJson(wrapper);
  expect(serialized).toMatchSnapshot();
});
