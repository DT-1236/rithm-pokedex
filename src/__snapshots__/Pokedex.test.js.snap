// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="d-flex flex-wrap"
>
  <Pokecard
    experience={62}
    id={4}
    name="Charmander"
    type="fire"
  />
  <Pokecard
    experience={63}
    id={7}
    name="Squirtle"
    type="water"
  />
  <Pokecard
    experience={72}
    id={11}
    name="Metapod"
    type="bug"
  />
  <Pokecard
    experience={178}
    id={12}
    name="Butterfree"
    type="flying"
  />
  <Pokecard
    experience={112}
    id={25}
    name="Pikachu"
    type="electric"
  />
  <Pokecard
    experience={95}
    id={39}
    name="Jigglypuff"
    type="normal"
  />
  <Pokecard
    experience={225}
    id={94}
    name="Gengar"
    type="poison"
  />
  <Pokecard
    experience={65}
    id={133}
    name="Eevee"
    type="normal"
  />
</div>
`;
